src/ansys/rocky/core/launcher.py

```diff
@@ -33,7 +32,6 @@
 from ansys.rocky.core.client import DEFAULT_SERVER_PORT, RockyClient, connect
 from ansys.rocky.core.exceptions import FreeflowLaunchError, RockyLaunchError
 
-_CONNECT_TO_SERVER_TIMEOUT = 60
```

Comment:
Should we keep this for backward compatibility?

Reply:
I don't think so. It was set as private for a reason.